var/da/da_radiance/da_read_obs_fy3c.inc

```diff
@@ -0,0 +1,801 @@
+   integer,parameter  :: num_low_freq_chan=15
+! interface variable
+   integer iret                         ! return status
+   integer(HID_T) fhnd1, fhnd2, fhnd3   ! file handle
+   integer(HID_T) ahnd1, ahnd2, ahnd3   ! attribute handle
+   integer(HID_T) dhnd1, dhnd2, dhnd3   ! dataset handle
+   integer(HID_T) shnd1, shnd2, shnd3   ! dataspace handle
+   integer(HSIZE_T) sz1(3)              ! array size 1
+   integer(HSIZE_T) sz2(3)              ! array size 2
+
+   integer(4) :: nscan                       ! NumberOfScans
+   real(4)    :: slp,itp                         ! ScaleFactor
+   real(8)    :: r8d1(max_scan)              ! array buffer for real(8) D1
+   integer(4) :: i4d2hi(hi_rez_fov,max_scan) ! array buffer for integer(4) D2 high
+
+   integer :: yr,mt,dy,hr,mn,sc
+   integer :: tess,tesp,allocstat
+   integer :: scnday(max_scan),scntim(max_scan)
+   real(8)    :: ju
+   integer,allocatable  :: date(:,:)
+   real(4),allocatable     :: tbb(:,:,:)
+
+! array data
+   real(4) :: latlr(lo_rez_fov,max_scan)   ! lat for low resolution
+   real(4) :: lonlr(lo_rez_fov,max_scan)   ! lon for low resolution
+
+   real(4) :: tb89ah(hi_rez_fov,max_scan)  ! tb for 89ah
+   real(4) :: tb89av(hi_rez_fov,max_scan)  ! tb for 89av
+
+!   real(4) :: tbb(98,2385,15)
+
+   integer(4) :: loflo(lo_rez_fov,max_scan) ! land ocean flag for low
+   integer(4) :: elev(lo_rez_fov,max_scan)  ! elevation
+
+   integer    :: satzen0(lo_rez_fov,max_scan)  ! satellite zenith
+   integer    :: satazi0(lo_rez_fov,max_scan)  ! satellite azimuth
+   integer    :: solzen0(lo_rez_fov,max_scan)  ! sun zenith
+   integer    :: solazi0(lo_rez_fov,max_scan)  ! sun azimuth
+
+   real    :: satzen(lo_rez_fov,max_scan)  ! satellite zenith
+   real    :: satazi(lo_rez_fov,max_scan)  ! satellite azimuth
+   real    :: solzen(lo_rez_fov,max_scan)  ! sun zenith
+   real    :: solazi(lo_rez_fov,max_scan)  ! sun azimuth
+
+   
+   real(r_kind)            :: R90    = 90.0_r_kind
+   real(r_kind),parameter  :: tbmin  = 50._r_kind
+   real(r_kind),parameter  :: tbmax  = 550._r_kind
+
+
+   real(kind=8)                   :: obs_time
+   type (datalink_type),pointer   :: head, p, current, prev
+   type(info_type)                :: info
+   type(model_loc_type)           :: loc
+
+   integer(i_kind)    :: idate5(6)
+
+   integer(i_kind)   :: inst,platform_id,satellite_id,sensor_id
+   real(r_kind)      :: tb, crit
+   integer(i_kind)   :: ifgat, iout, iobs
+   logical           :: outside, outside_all, iuse
+
+   integer           :: i,j,k,l,m,n, ifile, landsea_mask
+   logical           :: found, head_found, head_allocated
+
+! Other work variables
+   real(r_kind)     :: dlon_earth,dlat_earth
+   integer(i_kind)  :: num_amsr2_local, num_amsr2_global, num_amsr2_used, num_amsr2_thinned
+   integer(i_kind)  :: num_amsr2_used_tmp, num_amsr2_file
+   integer(i_kind)  :: num_amsr2_local_local, num_amsr2_global_local, num_amsr2_file_local
```

Comment:
Here and throughout, amsr2/AMSR2 needs to be changed to mwhs2/MWHS2.

Reply:
All the names have been changed according to your suggestion. Thanks.